.circleci/config.yml

```diff
@@ -27,6 +27,15 @@ parameters:
   det-version:
     type: string
     default: 0.17.6-dev0
+  docker-image:
+    type: string
+    default: cimg/base:2021.11
+  machine-image:
+    type: string
+    default: ubuntu-2004:202101-01
+  cache-buster:
```

Comment:
when/why would we update this cache-buster value?  Maybe that's worth a comment?